examples/api/service/host.go

```diff
@@ -32,7 +32,7 @@ func main() {
 	if err != nil {
 		panic(err)
 	}
-	fmt.Printf("I am peer: %s\n", nd.Identity)
+	fmt.Printf("I am peer: %s\n", nd.Identity.Pretty())
```

Comment:
Is this the right format we want to output? Seems there are multiple encoding formats available.